### DISCUSSION TITLE
Duplicate Logs in App Insights

### DISCUSSION THREAD

**Discussion by VictorioBerra** [Dotnet-Boxed/Templates#2028] [Q&A]
Is this because we are using both:

`AddApplicationInsightsTelemetry()` and Serilog `.WriteTo.Conditional( ... x => x.ApplicationInsights(`?

The docs say it automatically collects ILogger stuff: https://learn.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core?tabs=netcorenew%2Cnetcore6#ilogger-logs

Do I need to maybe just disable AppInsights logging?

```json
{
  "Logging": {
    "LogLevel": {
      "Default": "Information"
    },
    "ApplicationInsights": {
      "LogLevel": {
        "Default": "None"
      }
    }
  },
  "ApplicationInsights": {
    "ConnectionString": "InstrumentationKey=00000000-0000-0000-0000-000000000000"
  }
}
```

Would that work?

**Comment by RehanSaeed** [comment]
Its because of Open Telemetry and logging both saving the same info. There is some overlap.
